main.tf

```diff
@@ -31,13 +31,33 @@ resource "ibm_resource_tag" "cloud_monitoring_tag" {
   tag_type    = "access"
 }
 
+###############################################################################
+# Resource Key (Default Manager Key)
+###############################################################################
+
 resource "ibm_resource_key" "resource_key" {
-  name                 = var.manager_key_name
+  count                = var.disable_access_key_creation ? 0 : 1
```

Comment:
wont this be a breaking change without a moved block? Why did upgrade test not fail?

Reply:
@iamar7 I think this was a breaking change. See https://github.com/terraform-ibm-modules/terraform-ibm-cloud-monitoring/issues/120
Can you take a look please?